source/configure/elasticsearch-configuration-settings.rst

```diff
@@ -209,13 +216,37 @@ Bulk indexing
 | index and search results will be incomplete.                                                                                             |
 +---------------------------------------------------------------+--------------------------------------------------------------------------+
 
+.. config:setting:: elastic-indexestoskipwhilepurging
+  :displayname: Indexes to skip while purging (Elasticsearch)
+  :systemconsole: Environment > Elasticsearch
+  :configjson: .Elasticsearchsettings.IgnoredPurgeIndexes
+  :environment: MM_ELASTICSEARCHSETTINGS_IGNOREDPURGEINDEXES
+  :description: Specify index names to ignore while purging indexes, separated by commas.
+
+
+Indexes to skip while purging
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++---------------------------------------------------------------+--------------------------------------------------------------------------+
+| Specify index names to ignore while purging indexes.          | - System Config path: **Environment > Elasticsearch**                    |
+| Separate multiple index names with commas.                    | - ``config.json`` setting: ElasticsearchSettings.IgnoredPurgeIndexes     |
```

Comment:
@isacikgoz - Do we want to include details on using an asterisk (*) to avoid exact matches?

Reply:
Sure, it can help if they are using some patterns on indexes they don't want to purge.

Reply:
Thanks, @isacikgoz! This PR is now ready for your final review.